.github/workflows/gradle.yml

```diff
@@ -47,8 +47,8 @@ jobs:
         uses: addnab/docker-run-action@v3
         with:
           image: ${{ matrix.container }}
-          options: -v ${{ github.workspace }}:/work -w /work -e ARTIFACTORY_PUBLISH_USERNAME=${{ secrets.ARTIFACTORY_USERNAME }} -e ARTIFACTORY_PUBLISH_PASSWORD=${{ secrets.ARTIFACTORY_PASSWORD }} -e GITHUB_REF=$GITHUB_REF
-          run: df . && rm -f semicolon_delimited_script && ./gradlew build --build-cache -PbuildServer -PskipJavaFormat ${{ matrix.build-options }} ${{ env.EXTRA_GRADLE_ARGS }}
+          options: -v ${{ github.workspace }}:/work -w /work -e ARTIFACTORY_PUBLISH_USERNAME -e ARTIFACTORY_PUBLISH_PASSWORD -e GITHUB_REF
```

Comment:
Don't the artifactory variables still need to be set?

Reply:
Just doing testing at the moment.  -e should pass the outer environment variable as-is

Reply:
Secrets aren't environment variables though, I don't think

Reply:
Added as environment